src/Common/checkStackSize.cpp

```diff
@@ -22,6 +22,52 @@ namespace DB
 static thread_local void * stack_address = nullptr;
 static thread_local size_t max_stack_size = 0;
 
+/**
+ * @param address_ - stack address
```

Comment:
Sorry, but void** suggests that it either will modify the pointer or it is an array of pointers...